test/test_prototype_transforms_consistency.py

```diff
@@ -93,6 +87,7 @@ def parametrization(self):
         ],
         supports_pil=False,
         make_images_kwargs=dict(DEFAULT_MAKE_IMAGES_KWARGS, dtypes=[torch.float]),
+        removed_params=["inplace"],
```

Comment:
Open question if we want to remove the `inplace` parameter. That would be BC breaking without a proper deprecation cycle.

Reply:
We decided to put it back. See #6530.